uPortal-api/uPortal-api-platform-impl/src/main/java/org/apereo/portal/api/groups/EntityService.java

```diff
@@ -48,76 +49,112 @@ public static EntityService instance() {
     }
 
     public Set<Entity> search(String entityType, String searchTerm) {
+        if (!StringUtils.isEmpty(entityType) && !StringUtils.isEmpty(searchTerm)) {
```

Comment:
How about using isNotBlank(entityType) ? this way I can win Syntactic sugar and whitespace checking too :)